rsmt2d_test.go

```diff
@@ -86,8 +83,7 @@ func TestEdsRepairTwice(t *testing.T) {
 		shareSize int
 		codec     rsmt2d.Codec
 	}{
-		{"leopardFF8", bufferSize, rsmt2d.NewLeoRSFF8Codec()},
-		{"leopardFF16", bufferSize, rsmt2d.NewLeoRSFF16Codec()},
+		{"klauspost/leopard", bufferSize, rsmt2d.NewLeoRSCodec()},
```

Comment:
This is called `klauspost/leopard` here but `leopard` above?

Reply:
That is just the name of the test. But makes sense to make it consistent: https://github.com/celestiaorg/rsmt2d/pull/136/commits/1689a464eccea6c1f665f8e4612eac49640ecf2f